### PR TITLE
README: add link to #onemkl slack channel

### PR DESCRIPTION
**Pull request by mkrainiuk**
# Description

PR adds link to #onemkl slack channel in Contribution section

Partially implements https://github.com/uxlfoundation/open-source-working-group/issues/4

# Checklist

## All Submissions

- [x] Do all unit tests pass locally? Attach a log. N/A, readme changes only
- [x] Have you formatted the code using clang-format? N/A, readme changes only

### CODE REVIEW

**Review by sknepper — APPROVED** [review]
Looks great, thanks @mkrainiuk !

File: README.md [hunk 1]
```diff
@@ -539,13 +539,11 @@ The oneMKL Interfaces project is governed by the UXL Foundation and you can get
...
 You can also join the mailing lists for the [UXL Foundation](https://lists.uxlfoundation.org/g/main/subgroups) to be informed of when meetings are happening and receive the latest information and discussions.
 
-You can contribute to this project and also contribute to the specification for this project, read the [CONTRIBUTING](CONTRIBUTING.md) page for more information.
-
 ---
 
 ## Contributing
 
-See [CONTRIBUTING](CONTRIBUTING.md) for more information.
+You can contribute to this project and also contribute to the specification for this project, read the [CONTRIBUTING](CONTRIBUTING.md) page for more information. You can also contact oneMKL developers and maintainers via [UXL Foundation Slack](https://slack-invite.uxlfoundation.org/) using [#onemkl](https://uxlfoundation.slack.com/archives/onemkl) channel.
```
Comment (dnhsieh-intel) [inline README.md]: Maybe add a link to spec?
```suggestion
You can contribute to this project and also contribute to [the specification for this project](https://spec.oneapi.io/versions/latest/elements/oneMKL/source/index.html). Please read the [CONTRIBUTING](CONTRIBUTING.md) page for more information. You can also contact oneMKL developers and maintainers via [UXL Foundation Slack](https://slack-invite.uxlfoundation.org/) using [#onemkl](https://uxlfoundation.slack.com/archives/onemkl) channel.
```

**Review by dnhsieh-intel — APPROVED** [review]
Thank you! I have one minor suggestion.

From my experience, the line "Partially fixes <issue>" in the description will close the issue once this PR is merged. GitHub recognizes "fixes", but not "partially".

File: README.md (hunk 1 above)
Comment (mkrainiuk) [reply README.md]: Good idea, thank you!

**Pull request merged**